doc/api/child_process.md

```diff
@@ -209,8 +209,7 @@ encoding, `Buffer` objects will be passed to the callback instead.
 const { exec } = require('child_process');
 exec('cat *.js missing_file | wc -l', (error, stdout, stderr) => {
   if (error) {
-    console.error(`exec error: ${error}`);
-    return;
+    console.error(`exit(${error.code}): ${error}`);
```

Comment:
Needs a `return;` but... it's still a mostly unrelated change. Can you back it out or move it to a separate commit?